cmd/init_project.go

```diff
@@ -114,6 +116,8 @@ func (o *projectOptions) bindCmdlineFlags(cmd *cobra.Command) {
 		"defaults to the go package of the current working directory.")
 	cmd.Flags().StringVar(&o.project.Domain, "domain", "my.domain", "domain for groups")
 	cmd.Flags().StringVar(&o.project.Version, "project-version", project.Version2, "project version")
+
```

Comment:
```suggestion
```

Reply:
I left the blank line to separate from the projects args block. Is there any special reason to suggest this?